### DISCUSSION TITLE
Data not showing

### DISCUSSION THREAD

**Discussion by Nikhil23121998** [peridigm/peridigm#277] [Q&A]
my peridigm tensile_test example not showing any results after completion of simulation. it is showing the value of displacement and other parameters as zero. 

Is it because my peridigm is not installed correctly or there is some another reason?

Please help me to resolve this issue.

**Comment by djlittl** [comment]
It should show displacement in the loading direction.

Do you see progress printed to stdout as the simulation runs?

How many time steps are you seeing in your output file (the .e file)?
